src/tv2_afvd_showstyle/config-manifests.ts

```diff
@@ -253,17 +245,17 @@ export const gfxDesignTemplates: ConfigManifestEntry[] = [
 				rank: 0
 			},
 			{
-				id: 'INewsStyleColumn',
-				name: 'iNews Style Column',
-				description: 'The selected style',
+				id: 'INewsSkemaColumn',
+				name: 'iNews Skema Column',
+				description: 'The selected skema',
 				type: ConfigManifestEntryType.STRING,
 				required: false,
 				defaultVal: '',
 				rank: 1
 			},
 			{
 				id: 'VizTemplate',
-				name: 'GFX Template Name',
```

Comment:
I believe that wherever, 'Viz Template Name' appears, we were going to change it to 'GFX Template Name' instead, and have a more detailed description (diferent between Qbox and Gallery showstyles). At least this is the way it was done in the two other GFX tables.